noxfile.py

```diff
@@ -29,8 +29,10 @@
 import re
 import shutil
 import sys
+import tomllib
 
 import nox
+from packaging.requirements import Requirement
```

Comment:
It's cool to learn that this exists!

Reply:
Yeah, I just found this.  I was not looking forward to parsing the dependency strings, but I was happy to find this.